### PR TITLE
HLM-571|Display the QR Code details on the UI

### PR DESCRIPTION
**Pull request by vishwajeet-eGov**
@saiprakash-v @varadeth

### CODE REVIEW

File: verification_app/src/components/CertificateStatus/index.js [hunk 1]
```diff
@@ -52,6 +53,48 @@ export const CertificateStatus = ({certificateData, goBack}) => {
...
+    }
+    return destructuredData;
+}
+
+
+    const formatQRData = () => {
+        
+        displayData = destructure(processData);
+     
+    for( var key in displayData){
```
Comment (prasanna-egov) [inline verification_app/src/components/CertificateStatus/index.js:84]: Code needs to be formatted.

File: verification_app/src/components/CertificateStatus/index.js [hunk 2]
```diff
@@ -145,12 +188,12 @@ export const CertificateStatus = ({certificateData, goBack}) => {
                     {
                         isValid && <table className="mt-3">
                             {
-                                Object.keys(CertificateDetailsPaths).map((key, index) => {
-                                    const context = CertificateDetailsPaths[key];
-                                    return (
-                                        <tr key={index}>
-                                            <td className="pr-3">{t('certificate.'+key)}</td>
-                                            <td className="font-weight-bolder">{context.format(pathOr("NA", context.path, data))}</td>
+
```
Comment (prasanna-egov) [inline verification_app/src/components/CertificateStatus/index.js:191]: Proper formatting for this.

**Comment by varadeth** [comment]
this PR will be closed and I will raise a new PR after #32 is merged

**Comment by vishwajeet-eGov** [comment]
this PR will be closed and Tejash will raise a new PR after https://github.com/egovernments/divoc-ui-sample-implementation/pull/32 is merged

**Pull request closed without merging**